Projeto Físico/SubqueryTable.sql

```diff
@@ -0,0 +1,8 @@
+SELECT *
```

Comment:
Antes do início do código adicionar um comentário contendo uma descrição sobre o que essa consulta está fazendo!